src/GraphQL.NewtonsoftJson/StringExtensions.cs

```diff
@@ -1,119 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-
-namespace GraphQL.NewtonsoftJson
-{
-    /// <summary>
-    /// Provides extension methods to deserialize json strings into object dictionaries.
-    /// </summary>
-    public static class StringExtensions
```

Comment:
Did these methods moved to serializers?

Reply:
In a manner of speaking.  `FromJson` and `ToInputs` were removed.  `Deserialize` exists on `IGraphQLTextSerializer` as a replacement for both.